projects/NeRF-Det/nerfdet/nerf_utils/nerf_mlp.py

```diff
@@ -0,0 +1,310 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import functools
+import math
+from typing import Callable, Optional
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class MLP(nn.Module):
```

Comment:
Can we use `MLP` in mmcv?

Reply:
I think it can be modified, but it’s not necessary as the files in nerf_utils  are basically moved from the original code and basically unchanged.